src/backgroundQueue/sinks/memberBundleRequest.sink.spec.ts

```diff
@@ -107,58 +111,124 @@ describe('memberBundleIssuance', () => {
+          expect.objectContaining({
+            specversion: '1.0',
+          }),
+        );
+      });
+
+      test('Type should be outgoing service message', async () => {
         await postEvent(triggerEvent, server);
 
-        expect(mockPostToAwala).toHaveBeenCalledOnceWith(
-          requestBody,
-          expect.anything(),
-          expect.anything(),
+        expect(publishedEvents).toContainEqual(
+          expect.objectContaining({
+            type: OUTGOING_SERVICE_MESSAGE_TYPE,
+          }),
         );
       });
 
-      test('Should be called with correct Awala PDA', async () => {
+      test('Id should be member public key id', async () => {
         await postEvent(triggerEvent, server);
 
-        expect(mockPostToAwala).toHaveBeenCalledOnceWith(
-          expect.anything(),
-          AWALA_PDA,
-          expect.anything(),
+        expect(publishedEvents).toContainEqual(
+          expect.objectContaining({
+            id: MEMBER_PUBLIC_KEY_MONGO_ID,
+          }),
         );
       });
 
-      test('Should be called with correct Awala middleware endpoint', async () => {
+      test('Source should be awala endpoint internet', async () => {
         await postEvent(triggerEvent, server);
 
-        expect(mockPostToAwala).toHaveBeenCalledOnceWith(
-          expect.anything(),
-          expect.anything(),
-          new URL(AWALA_MIDDLEWARE_ENDPOINT),
+        expect(publishedEvents).toContainEqual(
+          expect.objectContaining({
+            source: OUTGOING_MESSAGE_SENDER_ID,
+          }),
+        );
+      });
+
+      test('Subject should be peer id', async () => {
+        await postEvent(triggerEvent, server);
+
+        expect(publishedEvents).toContainEqual(
+          expect.objectContaining({
+            subject: PEER_ID,
+          }),
+        );
+      });
+
+      test('Data content type should be member bundle', async () => {
+        await postEvent(triggerEvent, server);
+
+        expect(publishedEvents).toContainEqual(
+          expect.objectContaining({
+            datacontenttype: VeraidContentType.MEMBER_BUNDLE,
+          }),
+        );
+      });
+
+      test('Data should be buffer from member bundle', async () => {
+        await postEvent(triggerEvent, server);
+
+        expect(publishedEvents).toContainEqual(
+          expect.objectContaining({
+            data: Buffer.from(memberBundle),
+          }),
+        );
+      });
+
+      test('Time should be the time of creation of the message in iso format', async () => {
+        const creationDate = new Date();
+
+        await postEvent(triggerEvent, server);
+
+        expect(publishedEvents).toContainEqual(
+          expect.objectContaining({
+            time: expect.toSatisfy((time: string) => {
+              const dateFromTime = parseISO(time);
+              return creationDate <= dateFromTime && dateFromTime <= addSeconds(creationDate, 20);
+            }),
+          }),
+        );
```

Comment:
Nitpick: We avoid adding/subtracting seconds when we can simply initialise a date before the function call and another afterwards; e.g.:

```suggestion
        const postCreationDate = new Date();
        expect(publishedEvents).toContainEqual(
          expect.objectContaining({
            time: expect.toSatisfy((time: string) => {
              const dateFromTime = parseISO(time);
              return creationDate <= dateFromTime && dateFromTime <= postCreationDate;
            }),
          }),
        );
```